protobuf/drand/control.proto

```diff
@@ -256,18 +256,16 @@ message LoadBeaconResponse {
 }
 
 message StartFollowRequest {
-    // hex format
-    string info_hash = 1;

```

Comment:
why are we getting rid of this field / making this backwards-incompatible?

Reply:
Because we are not using the field anymore, and it's only for FollowRequest, so I thought I could just remove it.
Otherwise I can let it there and open an issue to remove it at some point in the future.